src/client.js

```diff
@@ -589,8 +592,19 @@ class Admin {
                 })
               }),
               pushNotifications: TODOnotImplemented,
-              users: TODOnotImplemented,
-              dev: TODOnotImplemented,
+              users: () => ({
+                list: (filter) => api._get(`${appUrl}/users`, filter),
+                create: (user) => api._post(`${appUrl}/users`, user),
+                user: TODOnotImplemented
+              }),
+              dev: () => ({
+                executePipeline: (body, userId, options) => {
```

Comment:
on a related note, the v1 version of this method accepted `options`, but I wasn't 100% sure of what those options actually are. Is this needed and/or even valid? cc @mpobrien

Reply:
It does get used in one additional place: https://github.com/10gen/stitch-ui/blob/105abcbe2965a42ab08c33688a6bbf7936d443af/src/push/actions.js#L25

For the push notifications page, we actually use the sandbox to do the sending of the push notifications. The option specified is just a query param that indicates to the server that it should run the pipeline as a system user and bypass rules.